lib/shipcloud/shipment_quote.rb

```diff
@@ -0,0 +1,24 @@
+module Shipcloud
+  class ShipmentQuote < Base
+    include Shipcloud::Operations::Create
+
+    attr_accessor :from, :to, :carrier, :package, :service
+    attr_reader :shipment_quote
+
+    def self.base_url
+      "#{camel_to_snakecase(class_name)}s"
+    end
+
+    def self.price
+      shipment_quote.price
+    end
+
+    def self.camel_to_snakecase(string)
+      string.gsub(/::/, "/").
+        gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2').
+        gsub(/([a-z\d])([A-Z])/, '\1_\2').
+        tr("-", "_").
+        downcase
+    end
```

Comment:
I would suggest to either move this to the Base class so it can be reused or (to keep it simple) just use the string "shipment_quote" in the .base_url method.